package.json

```diff
@@ -257,7 +257,7 @@
     "react-dom": "^18.1.0",
     "react-test-renderer": "^18.1.0",
     "semantic-release": "^19.0.5",
-    "stream-chat": "^8.46.1",
+    "stream-chat": "^8.47.1",
```

Comment:
Additionally, bump `stream-chat` version to incorporate recent `<StrictMode />`-related fixes in the client (https://github.com/GetStream/stream-chat-js/pull/1421).